project/EnsimeBuild.scala

```diff
@@ -28,7 +28,7 @@ object EnsimeBuild extends Build {
        "io.spray" %% "spray-json" % "1.3.2"
     ),
 
-    // disabling shared memory gives a small performance boost to tests
+      // disabling shared memory gives a small performance boost to tests
```

Comment:
Do you want me to fix it? Or leave it? If I fix it, we're going to wait for all of the tests to run again.